Create ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md (#2118)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1018-503-90-body-text-in-the-confirmation-modal-for-logging-out-all-users.md

---
date: ""

> title: Confirmation modal for logging out all users now translated correctly

Reviewer: ```suggestion
title: Confirmation message on logging out all users now translated correctly
```

> title: Body text in the confirmation modal for logging out all users is not translated (#6281) [GRAFT][release/y2024] (#6435)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Confirmation modal for logging out all users now has translated body text
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58016
version: 1018.503.90
---

> Body text in the confirmation modal for logging out all users is not translated (#6281) [GRAFT][release/y2024] (#6435)

Author: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT platform, there is a confirmation modal that appears when an administrator attempts to log out all users. Previously, the body text in this modal was not properly translated and always appeared in English, regardless of the user's language settings. With this change, the body text in the confirmation modal is now correctly translated based on the user's selected language. This improves the user experience for non-English speaking users by providing a fully localized interface when logging out all users.
```

> In the Cumulocity IoT platform, there is a confirmation modal that appears when an administrator attempts to log out all users. Previously, the body text in this modal was not properly translated and always appeared in English, regardless of the user's language settings. With this change, the body text in the confirmation modal is now correctly translated based on the user's selected language.

Reviewer: ```suggestion
In the Administration application, a confirmation modal appears when an administrator attempts to log out all users. Previously, the body text in this modal was not properly translated and always appeared in English, regardless of the user's language settings. With this change, the body text in the confirmation modal is now correctly translated based on the user's selected language.
```
